### PR TITLE
docs: python client

### PR DESCRIPTION
**Pull request by cprasad1**
### Description 
Python client docs generated using [pdoc3](https://pdoc3.github.io/pdoc/). To generate the docs, checkout the python client repo and run:

```
pdoc3 --html PATH_TO_PYTHON_CLIENT/ksqldb_confluent
```

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: docs/developer-guide/ksqldb-clients/python-client.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+---
+layout: page
+title: Python Client for ksqlDB
+tagline: Python client for ksqlDB
+description: Send requests to ksqlDB from your Python app
+keywords: ksqlDB, python, client
+---
+
+The Python clients enables sending requests easily to a ksqlDB server
```
Comment (JimGalasyn) [inline docs/developer-guide/ksqldb-clients/python-client.md:9]:
```suggestion
The Python client enables sending requests easily to a ksqlDB server
```

File: docs/developer-guide/ksqldb-clients/python-client.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+from within your Python application, as an alternative to using the [REST API](../api.md).
+The client only supports pull and push queries at the moment.
```
Comment (JimGalasyn) [inline docs/developer-guide/ksqldb-clients/python-client.md:11]:
```suggestion
The client only supports pull and push queries.
```

File: docs/developer-guide/ksqldb-clients/python-client.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+description: Send requests to ksqlDB from your Python app
+keywords: ksqlDB, python, client
+---
+
+The Python client enables sending requests easily to a ksqlDB server
+from within your Python application, as an alternative to using the [REST API](../api.md).
+The client only supports pull and push queries.
+
+!!! tip
+    [View the Python client API documentation](api/BLAHBLAHNEEDTOCHANGE.html)
```
Comment (cprasad1) [inline docs/developer-guide/ksqldb-clients/python-client.md:14]: @JimGalasyn How can I link this to [docs/python-client/html/ksqldb_confluent/index.html](https://github.com/confluentinc/ksql/pull/9265/files#diff-d6d8b766aa07f72826529755dca2cb1a5965ec92e81c029240b0af5a948c093a)?

**Review by JimGalasyn — APPROVED** [review]
LGTM!

**Pull request merged**